### PR TITLE
Final Blake2b PR. Merge this and close others.

### PR DESCRIPTION
**Pull request by MadeofTin**
I have consolidated all of the PRs for EIP-131 in order to preserve the change log. Please merge this EIP and close 

 * https://github.com/ethereum/EIPs/pull/2129
 * https://github.com/ethereum/EIPs/pull/152

### CODE REVIEW

File: EIPS/eip-131.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+---
+eip: 131
+title: Add Blake2 compression function `F` precompile
+author: Tjaden Hess <tah83@cornell.edu>, Matt Luongo (@mhluongo), Piotr Dyraga (@pdyraga), James Hancock (@madeoftin)
+discussions-to: https://ethereum-magicians.org/t/blake2b-f-precompile/3157
+status: Draft
+type: Standards Track
+category: Core
+created 2016-10-04
```
Comment (nicksavers) [inline EIPS/eip-131.md]: This is missing a colon after `created`

**Comment by axic** [comment]
Since this will be merged using the squash method, the changelog won't be visible.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Contains new file EIPS/eip-131.md
 - EIP 1679 requires approval from one of (@5chdn, @axic)

**Comment by holiman** [comment]
This one looks more up to date: https://github.com/ethereum/EIPs/pull/2129

**Comment by axic** [comment]
#2129 was merged. Closing this.

**Pull request closed without merging**